Resources/Scripts/main.js

```diff
@@ -10,6 +10,7 @@ import { SPIBuildLogNavigation } from './build_log_navigation.js'
 import { SPIAutofocus } from './autofocus.js'
 import { SPIPlaygroundsAppLinkFallback } from './playgrounds_app_link.js'
 import { SPIReadmeElement } from './readme_element.js'
+import { SPITabBarElement } from './package_tabs.js'
```

Comment:
Good rename of the class, but let's keep the filename in sync as `tab_bar_element.js`.